specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/virtualMachineExamples/VirtualMachine_Get.json

```diff
@@ -4,7 +4,6 @@
     "resourceGroupName": "myResourceGroup",
     "vmName": "myVM",
     "api-version": "2022-08-01",
-    "expand": "UserData"
```

Comment:
we were not supposed to remove userData and $expand from the generic VirtualMachines Get example. pls. add it back and change the expand to $expand.

Reply:
I must have misunderstood. I thought the intent was to either remove expan or add $

Reply:
If you remove $expand = userData from query then you also remove it from the response, any ways better to leave it to provide more details in the example.